tasks/buildimages.py

```diff
@@ -12,17 +12,22 @@
 
 @task(
     help={
-        "image_tag": "tag from build_image with format v<build_id>_<commit_id>",
-        "test_version": "Is a test image or not",
+        "tag": "tag from build_image with format v<build_id>_<commit_id>",
+        "images": "The image(s) to update, comma separated. If empty, updates the DATADOG_AGENT_ buildimages. Can be a incomplete pattern, e.g. 'deb,rpm' will update all deb and rpm images. deb_x64 will update only one image. Use `all` to update all images",
```

Comment:
I feel like `images` being a substring of the image names makes it super easy to change too many images...
If this will be called automatically by a script or by CI, then I expect it will explicitly list every image changed, not using that logic
Is there a case where you expect someone to use `deb` for example ? It would mean you changed all deb images and no other, which seems tricky to ensure

Reply:
Yes it was for a human to make it simpler to call when, for instance, you modify all `deb` or all `x64`. As you say we can consider
* if done automatically then the full list can be given
* buildimages can provide this list through the notification system
So I can change the logic to make it more strict (and then probably less error prone)